_posts/2024-07-15-openseeds-in-2024.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: 'Open Seeds in 2024: one cohort, two streams: OLS-9, and CAT-1'
+authors:
+- yochannah
+image: https://images.unsplash.com/photo-1486868018400-960892a825f4
+photos:
+  name: Nathan Anderson
+  license: CC BY-SA 4.0
+  url: https://images.unsplash.com/photo-1486868018400-960892a825f4
+---
+
+## How do we sustain open science and open research as a business? 
+
+2024 has been a year of change for OLS. Our original CZI core funding is winding up, with all four of the cohorts (OLS-5 to OLS-8) now completed. 
+
+We weren't too worried sustainability until at least 2026, through our [NASA grants]({% link _posts/2023-05-05-NASA-grant-announcement.md %}), which now support the [5-week Nebula program]({% link nebula/index.md %}). Once [OLS-8]({% link openseeds/ols-8/index.md%}) wrapped up, though, we weren't entirely sure how to continue running [Open Seeds]({% link openseeds/index.md %}), even though it was our original, core program. 
```

Comment:
```suggestion
We weren't too worried sustainability until at least 2026, through our [NASA grants]({% link _posts/2023-05-05-NASA-grant-announcement.md %}), which now support the [5-week Nebula program]({% link nebula/index.md %}). Once [OLS-8]({% link openseeds/ols-8/index.md %}) wrapped up, though, we weren't entirely sure how to continue running [Open Seeds]({% link openseeds/index.md %}), even though it was our original, core program. 
```

Reply:
@yochannah - Please read the first sentence of this paragraph. Think we're missing an "about" after **worried**?

Reply:
Resolved by Malvika's ingenuity.